src/notation/internal/notationconfiguration.h

```diff
@@ -89,11 +91,32 @@ class NotationConfiguration : public INotationConfiguration, public async::Async
     void setToolbarActions(const std::string& toolbarName, const std::vector<std::string>& actions) override;
 
     ValCh<bool> isNavigatorVisible() const override;
-    void setNavigatorVisible(bool visible) override;
+    void setIsNavigatorVisible(bool visible) override;
 
     ValCh<framework::Orientation> canvasOrientation() const override;
     void setCanvasOrientation(framework::Orientation orientation) override;
 
+    ValCh<bool> isPalettePanelVisible() const override;
```

Comment:
I believe you should merge all those isSomethingVisible methods in one

Reply:
thanks, fixed